### DISCUSSION TITLE
Disabling try-now functionality

### DISCUSSION THREAD

**Discussion by ldrahan** [eclipse-theia/theia-cloud#350] [Q&A] [answered]
Hello

I'm going to create an API that will start a session using the endpoint _servicex/service/session_.
Is there a way to disable starting session using  _/trynow/_? In that way starting new session will be available only using the API

**Comment by sgraband** [comment, marked as answer]
Hey @ldrahan,

Currently there is no flag that let's you disable the try-now page all together. I think you have three options:

1. Use a landing page, that simply does nothing (e.g. some random example page with no functionality)
2. Delete all the landing page related resources after applying the helm chart:
3. Checkout the helm repository and adapt the helm chart

I think a flag to disable the creation of the landing-page makes sense (like with the demoApplication). If you work on this and would want to contribute this, we would gladly review and accept the contribution. Hope this helps.

**Reply by ldrahan** [reply]
@sgraband thank you for the answer. I will try the first option
